MDANSE_GUI/Src/MDANSE_GUI/Tabs/Plotters/Plotter.py

```diff
@@ -278,3 +278,16 @@ def plot(
         axes = target.add_subplot(111)
         self._axes = [axes]
         self.apply_settings(plotting_context)
+
+    def plot_blank(self, draw_cross: bool = True):
+        """Inform the user that no data could be plotted."""
+        figure = self.get_figure()
+        axes = figure.add_subplot(111)
+        if draw_cross:
+            axes.plot([0, 1], [0, 1], "k-")
+            axes.plot([0, 1], [1, 0], "k-")
```

Comment:
```suggestion
            axes.axline([0, 1], [0, 1], color="k", linestyle="-")
            axes.axline([0, 1], [1, 0], color="k", linestyle="-")
```
For infinitely long lines. More obvious they're not data?